.github/workflows/go.yml

```
@@ -35,17 +35,24 @@ jobs:
         mkdir --parents /home/runner/go/src/github.com/vertgenlab/ &&
         cp -r ../gonomics /home/runner/go/src/github.com/vertgenlab/ &&
         cd /home/runner/go/src/github.com/vertgenlab/gonomics &&
+        go install ./... &&
         go test ./...
         
   format:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
         with:
           ref: ${{ github.head_ref }}
       - uses: actions/setup-go@v3
```

actions/checkout@v3 will force you to upgrade to [setup-go@v4](https://github.com/actions/setup-go#v4)
```suggestion
      - uses: actions/setup-go@v4
```